src/pages/home/sidebar/SidebarScreen.js

```diff
@@ -170,5 +180,8 @@ export default compose(
         betas: {
             key: ONYXKEYS.BETAS,
         },
+        isNewUser: {
```

Comment:
Please rename this variable to something like `isFirstTimeNewExpensifyUser`, and add it to `propTypes` at the top of this file, with a comment